set SCS-0004 certificate validity period to one year (#1168)
SovereignCloudStack/standards: Standards/scs-0004-v1-achieving-certification.md

0. Certificates are issued by the SCS certification assessment body. This is either the _Forum SCS-Standards_, part of the Open Source Business Alliance e.V. (OSBA), or an organization appointed by the _Forum SCS-Standards_ . An interested party has to apply for certification with this body, which in turn determines the rules that govern what parties are eligible for application (fees may apply).

> 1. Each certificate issued pertains to a given combination of subject (specific cloud environment including IaaS regions or KaaS configurations), scope (such as _SCS-compatible IaaS_ or _SCS-compatible KaaS_), and version of that scope. The certificate is only valid for that combination for the period of one year. The certificate remains valid for this period, even if the scope version expires for which it applies.

Reviewer: ```suggestion
1. Each certificate issued pertains to a given combination of subject (specific cloud environment including IaaS regions or KaaS configurations), scope (such as _SCS-compatible IaaS_ or _SCS-compatible KaaS_), and version of that scope. The certificate is only valid for that combination for the period of one year (regardless of the expiration date of the scope version).
```
